<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessAnalyToolWidget</class>
 <widget class="ToolWidget" name="AccessAnalyToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AccessAnalyToolWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_demond" stretch="10,0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_demand">
       <item>
        <widget class="QLabel" name="label_demand">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Choose demand layer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_demand"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_btndema" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>23</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_demand">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_supply">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,3">
       <item>
        <widget class="QLabel" name="label_supply">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Choose supply layers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_supply">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_road">
       <item>
        <widget class="QLabel" name="label_road">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Choose road layer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_road"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_btnroad" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>23</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_demand_2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_btn">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_help">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolWidget</class>
   <extends>QWidget</extends>
   <header>toolwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
